mockdata/src/main/resources/venuse_tbd.txt

```diff
@@ -0,0 +1,13 @@
+Tower Theater
+The Fillmore
+Fenway Park,Boston, MA, 37000
+Barclay's Center,Brooklyn,NY,19000
+The Staples Center,Los Angeles, CA,XX,20000
+Carnegie Hall,New York, NY,2800
+Madison Square Garden,New York, NY,20000
+The Gorge Amphitheater,Grant County,WA, 30000
+The Greek Theater, Los Angeles, CA
+Ottobar, Baltimore, MD
+Radio City Music Hall
+Grand Ole Opry House
+The Bluebird Cafe,Nashville, TN
```

Comment:
ALL THE VENUES 
![image](https://github.com/schroedermatt/data-demo/assets/3487825/c571c606-0e38-46f1-958c-3f6810fdeb9a)